git-storage/src/backend/read.rs

```diff
@@ -46,6 +46,14 @@ impl Read {
     }
 }
 
+impl core::fmt::Debug for Read {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.debug_struct("Read")
+            .field("raw", &self.raw.path())
+            .finish()
+    }
+}
```

Comment:
nit: I prefer importing `std::fmt`
```suggestion
impl fmt::Debug for Read {
    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
        f.debug_struct("Read")
            .field("raw", &self.raw.path())
            .finish()
    }
}
```

Reply:
Ah sure, force of habit for no-std!